osx/setup.sh

```diff
@@ -363,8 +364,19 @@ fi
 
 # Remove the .dmg if it exists.
 test -e "$DMG_PATH" && rm "$DMG_PATH"
+
+if [ -d "$RELEASE_PATH/More files" ]; then
```

Comment:
@amodia Change this into a "constant" -> `"$RELEASE_PATH/More files"` so it's easy to maintain.